extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -164,8 +162,14 @@ public BitmapFont generateFont (FreeTypeFontParameter parameter) {
 	 * @param parameter configures how the font is generated */
 	public BitmapFont generateFont (FreeTypeFontParameter parameter, FreeTypeBitmapFontData data) {

```

Comment:
I'm not sure - my feeling is that the generator should be the class to generate fonts, not the FreeTypeBitmapFontData class.  You have a more comprehensive understanding of the code, though, so your intuitions are probably better :)